app/tests/test_collections_endpoint.py

```diff
@@ -84,3 +118,164 @@ def test_collections_limit_query(self):
 
         response = self.client.get(f"/{API_BASE}/collections?limit=1000")
         self.assertEqual(400, response.status_code)
+
+    def test_valid_collections_post(self):
+        payload_json = self.sample_collections['valid_collection_set_1']
+        grant_permissions('Collection', self.user)
+        self.client.login(username=self.username, password=self.password)
+
+        response = self.client.post(
+            f"/{API_BASE}/collections",
+            data=json.dumps(payload_json),
```

Comment:
For some reason I had to use `json.dumps()` here, although the same test seems to work for @rebert in https://github.com/geoadmin/service-stac/pull/132 without it.
When I omit `json.dumps()` here, there would be an issue with single instead of double quotes for the json-payload..